TurtleArtActivity.py

```diff
@@ -191,7 +192,7 @@ def check_buttons_for_fit(self):
         self._toolbox.toolbar.remove(self.stop_button)
         self._view_toolbar.remove(self._coordinates_toolitem)
 
-        if gtk.gdk.screen_width() / 14 < style.GRID_CELL_SIZE:
+        if Gtk.Gdk.screen_width() / 14 < style.GRID_CELL_SIZE:
```

Comment:
`Gdk` is independent of `Gtk` unlike in `pygtk2`
Also `Gdk` does not have a `screen_width` method. Instead you may use `Gdk.Screen.width()`